web/client/epics/playback.js

```diff
@@ -272,13 +272,11 @@ module.exports = {
     playbackStopWhenDeleteLayer: (action$, { getState = () => {} }= {}) =>
         action$
         .ofType(REMOVE_NODE)
-        .switchMap( action =>
-            selectedLayerSelector(getState()) === action.node || selectedLayerSelector(getState()) === ""
-            ? Rx.Observable.of(
-                stop()
-            ) :
-            Rx.Observable.empty()
-            )
+        .filter( () =>
+                !selectedLayerSelector(getState())
```

Comment:
I'd like more a check like : `selectedLayerSelector(getState()) === action.node`. Did this feature cause any problem?

Reply:
does the test in dimension-test.js line 46 fulfill this check ?